core/controllers/contributor_dashboard.py

```diff
@@ -283,6 +283,7 @@ class ReviewableOpportunitiesHandlerNormalizedRequestDict(TypedDict):
     """
 
     topic_name: Optional[str]
+    language: str
```

Comment:
```suggestion
    language_code: str
```

Ditto everywhere else.